### ISSUE TITLE
Get a list of biggest files in the repo

### ISSUE THREAD

**Issue by masics** [github/git-sizer#60]
It would be nice to get a list of N biggest files in the repo. Both compressed and uncompressed sizes to be shown.

**Comment by larsxschneider** [comment]
@masics I agree. In the meantime you can use this script to generate the output you want: https://github.com/github/platform-samples/blob/master/scripts/git-find-large-files

**Comment by ymartin59** [comment]
An alternative is `git rev-list --all | xargs -rL1 git ls-tree -r --long | sort -uk3 | sort -rnk4` but it may require large temporary storage...
